static/sass/_patterns_carousel.scss

```diff
@@ -25,58 +26,106 @@
       text-align: center;
 
       img,
-      video,
-      iframe {
+      video {
         display: block;
-        margin-bottom: 0;
+        margin-bottom: 0 !important;
```

Comment:
Why `!important`? If this is required to override something (third party styles?) we should have a comment here about it.

Reply:
Updated!